Dockerfile

```diff
@@ -0,0 +1,51 @@
+FROM ocaml/opam:debian-10-ocaml-4.11
+
+LABEL maintainer="gracewgao"
+
+SHELL ["/bin/bash", "-c"]
+
+# updates environment variables
+ENV PYRE_BINARY='/home/opam/pyre-check/source/_build/default/main.exe'
+ENV PYRE_TYPESHED='/home/opam/pyre-check/stubs/typeshed/typeshed-master'
+ENV PYTHONPATH='/home/opam/pyre-check:$PYTHONPATH'
+ENV PATH='/home/opam/.opam/4.10.2/bin:/home/opam/pyre-check/scripts:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:$PATH'
+
+RUN echo -e 'export PYRE_BINARY=/home/opam/pyre-check/source/_build/default/main.exe \n \
```

Comment:
Why do we need to both set these using ENV and put them in `.bashrc`? Shouldn't it be one or the other? If for some reason there is a process which doesn't inherit the variables through `ENV` but does pick them up from `.bashrc`, couldn't we just eliminate the duplicate `ENV` commands by moving to `RUN source ~/.bashrc`?

Reply:
I tried that first but it wasn't working, so I found some way online to define the environment variables in Docker using ENV - I added them into .bashrc afterwards because I figured they would need to be reevaluated every time they restart the container?

Reply:
It looks like variables set using `ENV` are supposed to persist when the container is run:
https://docs.docker.com/engine/reference/builder/#env